docs/en/operations/troubleshooting.md

```diff
@@ -0,0 +1,137 @@
+2019.01.11 15:25:11.156716 [ 2 ] {} <Information> BaseDaemon: Stop SignalListener thread
+```
+
+**See system.d logs**
+
+If there is no any useful information in `clickhouse-server` logs or there is no any logs, you can see `system.d` logs by the command:
+
+```
+sudo journalctl -u clickhouse-server
+```
+
+**Start clickhouse-server in interactive mode**
+
+```
+sudo -u clickhouse /usr/bin/clickhouse-server --config-file /etc/clickhouse-server/config.xml
+```
+
+This command starts the server as an interactive app with standard parameters of autostart script. In this mode `clickhouse-server` prints all the event messages into the console.
+
+### Configuration parameters
+
+Check:
+
+- Docker settings.
+
+    If you run ClickHouse in Docker in IPv6 network, check that `network=host` is set.
+
+- Endpoint settings.
+
+    Check [listen_host](server_settings/settings.md#server_settings-listen_host) and [tcp_port](server_settings/settings.md#server_settings-tcp_port) settings.
+
+    ClickHouse server accepts localhost connections only by default.
+
+- HTTP protocol settings.
+
+    Check protocol settings for HTTP API.
+
+- Secure connection settings.
+
+    Check setting `tcp_port_secure` for secure connection. Use proper parameters while connecting. For example, use parameter `port_secure` with `clickhouse_client`. Check also settings for ssl-sertificates.
+
+- User settings
+
+    You may use the wrong user name or password for it.
+
+## ClickHouse does not process queries {#troubleshooting-does-not-process-queries}
+
+If ClickHouse can not process the query, it sends the description of an error to the client. In the `clickhouse-client` you get a description of an error in console. If you use HTTP interface, ClickHouse sends error description in response body. For example,
+
+```
+$ curl 'http://localhost:8123/' --data-binary "SELECT a"
+Code: 47, e.displayText() = DB::Exception: Unknown identifier: a. Note that there is no tables (FROM clause) in your query, context: required_names: 'a' source_tables: table_aliases: private_aliases: column_aliases: public_columns: 'a' masked_columns: array_join_columns: source_columns: , e.what() = DB::Exception
+```
+
+If you start `clickhouse-client` with `stack-trace` parameter, ClickHouse returns server stack trace with the description of an error.
+
+It is possible that you see the message of connection broken. In this case, you can repeat query. If connection brakes any time you perform the query you should check the server logs for errors.
+
+## ClickHouse processes queries too slow {#troubleshooting-too-slow}
+
+If you see that ClickHouse works too slow, you need to profile the load of the server resources and network for your queries.
+
+You can use clickhouse-benchmark utility to profile queries. It shows the number of queries processed in a second, the number of rows processed in a second and percentiles of query processing times.
```

Comment:
`clickhouse-benchmark` -> link

Reply:
Unfortunately it will be link to nowere. I'll create a ticket for clickhouse-benchmark description.